src/components/Frame/components/Loading/Loading.tsx

```diff
@@ -1,78 +1,61 @@
-import React from 'react';
-import debounce from 'lodash/debounce';
+import React, {useEffect, useState} from 'react';
 import styles from './Loading.scss';
 
-export interface LoadingProps {}
-
-interface State {
-  progress: number;
-  step: number;
-  animation: number | null;
-}
-
 const INITIAL_STEP = 10;
 const STUCK_THRESHOLD = 99;
 
-export class Loading extends React.Component<LoadingProps, State> {
-  state: State = {
-    progress: 0,
-    step: INITIAL_STEP,
-    animation: null,
-  };
-
-  private ariaValuenow = debounce(() => {
-    const {progress} = this.state;
-    return Math.floor(progress / 10) * 10;
-  }, 15);
+export interface LoadingProps {}
 
-  componentDidMount() {
-    this.increment();
-  }
+export function Loading(_props: LoadingProps) {
```

Comment:
Andrew: I've got the horrible feeling that this exists because not having a `LoadingProps` type might do ugly things to the types finder in the styleguide. Could you give that a check?

Reply:
Yikes, I thought we used regex to find `{FILE_NAME}Props`. Do we use the ts compiler? I'll take a look and see how it works out

Reply:
Ahh this is `components/Frame/components/Loading` not `components/Loading`. This component doesn't show in the style guide so we can remove `export interface LoadingProps {}`